Tests/MongoSwiftTests/MongoCollection+BulkWriteTests.swift

```diff
@@ -0,0 +1,73 @@
+@testable import MongoSwift
+import Nimble
+import XCTest
+
+final class MongoCollection_BulkWriteTests: XCTestCase {
+    static var allTests: [(String, (MongoCollection_BulkWriteTests) -> () throws -> Void)] {
+        return [
+            ("testEmptyRequests", testEmptyRequests),
+            ("testUnsupportedWriteModel", testUnsupportedWriteModel)
+        ]
+    }
+
+    static var client: MongoClient?
+    var coll: MongoCollection<Document>!
+
+    /// Set up the entire suite - run once before all tests
+    override class func setUp() {
+        super.setUp()
+        do {
+            client = try MongoClient()
+        } catch {
+            print("Setup failed: \(error)")
+        }
+    }
+
+    /// Set up a single test - run before each testX function
+    override func setUp() {
+        super.setUp()
+
+        guard let client = MongoCollection_BulkWriteTests.client else {
+            return XCTFail("Client is not initialized")
+        }
+
+        let collectionName = String(describing: self)
+
+        do {
+            coll = try client.db("collectionTest").collection(collectionName)
+        } catch {
+            return XCTFail("Setup failed: \(error)")
+        }
+    }
+
+    /// Teardown a single test - run after each testX function
+    override func tearDown() {
+        do {
+            try coll.drop()
+        } catch {
+            /* TODO: Only ignore "ns not found" error once we can differentiate
+             * MongoError.commandError by error code. */
```

Comment:
👍

Reply:
I don't believe this is covered by any spec, but would it make sense to swallow this error in the Collection drop method? Then we wouldn't have to worry about them in tests at all. For reference, PHP does this in [DropCollection](https://github.com/mongodb/mongo-php-library/blob/1.4.2/src/Operation/DropCollection.php#L107).

I think this is desirable unless there's a reason users might want to see this error for themselves. It may be good to ask @ajdavis why libmongoc surfaces that error (I'm not aware of any other drivers that do, and the shell certainly does not).

AFAIK, this isn't an issue with dropping a database (server doesn't care if it exists or not).

Reply:
Yeah, PyMongo swallows "ns not found" also; I think the C Driver is eccentric here. But since it isn't in any spec, I'm not inclined to change the C Driver's behavior. I'll update the mongoc_collection_drop_with_opts docs.

Reply:
Created [SWIF-204](https://jira.mongodb.org/browse/SWIFT-204) to track this feature.